nflow-engine/src/test/java/io/nflow/engine/internal/dao/WorkflowInstanceDaoTest.java

```diff
@@ -454,7 +465,7 @@ public void processRow(ResultSet rs) throws SQLException {
   public void updateNotRunningWorkflowInstanceDoesNotUpdateRunningInstance() {
     WorkflowInstance instance = constructWorkflowInstanceBuilder().build();
     long id = dao.insertWorkflowInstance(instance);
-    assertThat(jdbc.update("update nflow_workflow set executor_id = ? where id = ?", executorDao.getExecutorId(), id), is(1));
+    setInstanceExecutorId(id);
     final DateTime tomorrow = now().plusDays(1);
```

Comment:
should this test (or a new one?) assert that no action was inserted?

Reply:
`updateNotRunningWorkflowInstance` is not affected by this PR.